src/concrete/ml/quantization/quantized_ops.py

```diff
@@ -219,18 +222,90 @@ def q_impl(
 
         p = weights_q_values.shape[0]
 
+        def enc_mul(x, y):
+            """Encrypted multiplication trick.
```

Comment:
it's not a trick when it's in production :) I don't see anything tricky about it anyway, let's use a more formal name

Reply:
I agree.

Reply:
Changed it.